pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1042,7 +1042,11 @@ type VSphereCloudSpec struct {
 	// +optional
 	Password string `json:"password"`
 	// VMNetName is the name of the vSphere network.
-	VMNetName string `json:"vmNetName"`
+	// +optional
+	VMNetName string `json:"vmNetName,omitempty"`
```

Comment:
If we're sure this is going to be deprecated, maybe a message in **the future tense** makes sense. For example, `VMNetName is going to be deprecated starting with KKP 2.24 and removed in KKP 2.26+`.